README.md

```diff
@@ -71,7 +71,8 @@ The following resources are available:
 * Pages: https://2017.drupalcamp.es/jsonapi/node/page
 * Sponsors: https://2017.drupalcamp.es/jsonapi/node/sponsor
 * Speakers: https://2017.drupalcamp.es/jsonapi/node/speaker
-* Sessions: https://2017.drupalcamp.es/sessions/proposed?_format=json
-* Session detail: https://2017.drupalcamp.es/sessions/vue-meets-drupal?_format=json
+* Proposed Sessions: https://2017.drupalcamp.es/sessions/proposed?_format=json
+* Selected Sessions: https://2017.drupalcamp.es/sessions/selected?_format=json
```

Comment:
@ccamposfuentes I have updated the README so now there is a resource to list proposed sessions and another one for the list of selected sessions.